input.tsv

```diff
@@ -0,0 +1,3 @@
+sampleID	read
+NA12877_R1_10k	/scratch/er01/cl9310/bio-test-datasets/fastq/NA12877_R1_10k.fq.gz
```

Comment:
The sample ID should be NA12877 for both lines. In its current form, this creates separate folders per fastq, not per sample.